src/Components/Catalog/Catalog.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react'
+import PropTypes from "prop-types"
+import Item from "./Item"
+
+export default class Catalog extends Component {
+  render() {
+    let items = []
+    this.props.items.forEach(item => {

```

Comment:
use map here 

```
{this.props.items.map(item => (
  <Item id={item._id}
    name={item.name}
    price={item.price}
    rating={item.avgRating}
    isOnSale={item.isOnSale}
    image={item.imageUrl}
    key={item._id} />
))}
```

instead of creating a new array

Reply:
Yeah map is a lot cleaner